static/js/click_handlers.js

```diff
@@ -218,13 +218,18 @@ export function initialize() {
         message_flags.toggle_starred_and_update_server(message);
     });
 
-    $("#main_div").on("click", ".message_reaction", function (e) {
+    $("#main_div").on("click", ".message_reaction", async function (e) {
         e.stopPropagation();
+        if ($(this).hasClass("update_state")) {
+            return;
+        }
```

Comment:
I'm not a good reviewer of front-end stuff, but I don't _think_ we currently use this pattern to prevent double-clicks.